learing_journal/security.py

```diff
@@ -0,0 +1,53 @@
+import os
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+from pyramid.security import Everyone, Authenticated
+from pyramid.security import Allow
+from passlib.apps import custom_app_context as pwd_context
+# https://pythonhosted.org/passlib/new_app_quickstart.html
+from pyramid.session import SignedCookieSessionFactory
+
+
+class MyRoot(object):
+
+    def __init__(self, request):
+        self.request = request
+
+    __acl__ = [
+        (Allow, Everyone, 'view'),
+        (Allow, Authenticated, 'secret'),
+    ]
+
+
+def includeme(config):
```

Comment:
I would prefer the `includeme` function to be the last thing in a file.  It makes it easier to see